history.txt

```diff
@@ -1,6 +1,39 @@
 OMERO version history
 =====================
 
+5.1.4 (September 2015)
+----------------------
+
+A bug-fix release covering all components. Improvements include:
+
+-  channel buttons fixed in OMERO.web
```

Comment:
What does "channel buttons fixed in OMERO.web" mean? They were working before weren't they?